source/_docs/installation/raspberry-pi.markdown

```diff
@@ -107,3 +107,11 @@ $ pip3 install --upgrade homeassistant
 ```
 
 Once the last command executes restart the Home Assistant service to apply the latest updates.  Please keep in mind that some updates may take longer to boot up than others.  If Home Assistant fails to start make sure you check the **Breaking Changes** from the [Release Notes](https://github.com/home-assistant/home-assistant/releases).
+
+### {% linkable_title Activating the virtual environment %}
+
+When instructions tell you to activate the virtual environment, you will use the following commands:
```

Comment:
`you will use the following commands`:

Maybe this is a little bit to commanding?